server/controller/tripRoutes.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const router = express.Router();
+const middleware = require('../middleware');
+const tripHandler = require('../handlers/tripHandler');
+
+// All the routes for trips
+router.get('/get', middleware.checkToken, tripHandler.getTrips);
+router.post('/add', middleware.checkToken, tripHandler.addTrip);
+router.put('/update/:tid', middleware.checkToken, tripHandler.updateTrip);
```

Comment:
Should we remove the :tid here?

Reply:
Yes. Good catch.

Reply:
We should probably take the :tid out before we merge!

Reply:
Whoops, yes!